bip-0003.md

```diff
@@ -596,13 +596,13 @@ The licenses of existing BIPs remain untouched.
 ## Copyright
 
 This BIP is licensed under the [BSD-2-Clause License](https://opensource.org/licenses/BSD-2-Clause). Some content was
-adapted from [BIP 2](BIP-0002.mediawiki) which was also licensed under the BSD-2-Clause.
+adapted from [BIP 2](bip-0002.mediawiki) which was also licensed under the BSD-2-Clause.
 
 ## Related Work
 
 - [BIP 1: BIP Purpose and Guidelines](bip-0001.mediawiki)
-- [BIP 2: BIP Process, revised](BIP-0002.mediawiki)
-- [BIP 123: BIP Classification](BIP-0123.mediawiki)
+- [BIP 2: BIP Process, revised](bip-0002.mediawiki)
+- [BIP 123: BIP Classification](bip-0123.mediawiki)
```

Comment:
Line 564 also needs updating for the url to BIP123.

Reply:
Hah, I was just reviewing the same PR. :)